lib/hca/enrollment_eligibility/service.rb

```diff
@@ -54,9 +54,9 @@ class Service < Common::Client::Base
 
       MEDICARE = 'Medicare'
 
-      def get_ezr_data(icn)
+      def get_ezr_data(user)
```

Comment:
Made this change so that we can just call the user of the flipper toggle instead of having to make another Active Record call.